stdlib/public/core/RandomAccessCollection.swift

```diff
@@ -269,9 +254,6 @@ where Index : Strideable,
   ///   If `n` is negative, this is the same value as the result of `-n` calls
   ///   to `index(before:)`.
   ///
-  /// - Precondition:
-  ///   - If `n > 0`, `n <= self.distance(from: i, to: self.endIndex)`
-  ///   - If `n < 0`, `n >= self.distance(from: i, to: self.startIndex)`
```

Comment:
I'll admit that the above line is 100% more clear than this goop, but shouldn't the line above be listed as a "precondition"?